File: go/reverse-string/reverse_string.go
package reverse

func Reverse(input string) string {
	runeChar := []rune(input)

	for i, j := 0, len(runeChar)-1; i < j; i, j = i+1, j-1 {
		runeChar[i], runeChar[j] = runeChar[j], runeChar[i]
	}

	return string(runeChar)
}

File: csharp/reverse-string/ReverseString.cs
using System;
using System.Linq;

public static class ReverseString
{
    public static string Reverse(string input) => new string(input.Reverse().ToArray());
    /*
    string reverseString = "";
    for (int i = input.Length - 1; i >= 0; i--)
    {
        reverseString += input[i];
    }
    return reverseString;
    */
}
